src/components/animated/_shake.less

```diff
@@ -3,6 +3,10 @@
     animation: shake 0.75s;
   }
 
+  &.animated {
+    animation: shake 0.75s infinite;
```

Comment:
Instead of repeating this line, can you please add `&.animated` to `&:hover` above and the just add a rule for `animation-iteration-count`?

Reply:
thanks, I've removed the duplicate lines